src/content/en/updates/2017/03/custom-scrollbar.md

```diff
@@ -0,0 +1,473 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Custom transforms matrices allow you to build frame-perfect custom scrollbars.
+
+{# wf_updated_on: 2017-03-23 #}
+{# wf_published_on: 2017-03-23 #}
+{# wf_tags: performance #}
+{# wf_featured_image: /web/updates/images/2017/03/custom-scrollbar/poster.jpg #}
+{# wf_featured_snippet: Custom transforms matrices allow you to build frame-perfect custom scrollbars. #}
+
+
+# CSS deep-dive: matrix3d() for a frame-perfect custom scrollbar {: .page-title }
+
+{% include "web/_shared/contributors/surma.html" %}
+
+<!-- HTML comments in this article contain LaTeX code that can be rendered to
```

Comment:
Nit - When rendered in DevSite, this comment doesn't occur until almost line 700. Might be worth putting that into some other place. :/